packages/now-node/src/types.ts

```diff
@@ -15,3 +15,8 @@ export type NowResponse = ServerResponse & {
   json: (jsonBody: any) => NowResponse;
   status: (statusCode: number) => NowResponse;
 };
+
+export type NowApiHandler = (
+  req: NowRequest,
+  res: NowResponse
+) => void;
```

Comment:
Can you add this usage to the `test/fixtures/15-helpers/ts/index.ts` file or create a new file `test/fixtures/15-helpers/ts/handler.ts`?

Reply:
would that be as simple as this?
```ts
import { NowApiHandler } from './types';

const listener: NowApiHandler = (req, res) => {
  res.status(200);
  res.send('hello:RANDOMNESS_PLACEHOLDER');
};

export default listener;

```